tests/endpoints/test_upload_endpoint_pvc.py

```
@@ -415,6 +425,162 @@ def test_upload_gaussian_data(self):
+        response = self.client.post(
+            "/data/upload",
+            content=compressed_payload,
+            headers={
+                "Content-Type": "application/json",
+                "Content-Encoding": "gzip",
+            },
+        )
+
+        # Debug output if test fails
+        if response.status_code != 200:
+            print(f"\n=== DEBUG INFO ===")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)